### PR TITLE
Add Bevy 0.11 support

### LINKED ISSUE #62: Bevy 0.11 Support

### PR DESCRIPTION
**Pull request by makspll**
WIP fixes #62

### CODE REVIEW

**Comment by zwazel** [comment]
Hi, wanted to know what the current state with this is?

**Comment by makspll** [comment]
Hi, sorry for late reply, I believe this branch should be fully functional, I didn't merge as I didn't have time to do final checks and pipeline fixes

File: examples/wrappers.rs [hunk 1]
```diff
@@ -36,7 +33,7 @@ impl MyThing {
 // Script references can also be made to subfields (even non reflectable ones) of types via sub reflection
 //
 // Note: this step is not fully necessary, if your value is reflectable, you'll be able to reach it via
-// The bevy API, however doing this means your provide static typing for your scripts in languages which support it,
+// The bevy API, however doing this means your type will have  provide static typing for your scripts in languages which support it,
```
Comment (LaserWitch) [inline examples/wrappers.rs:36]: I think you accidentally a word here

Comment (makspll) [reply examples/wrappers.rs]: yup!

**Pull request merged**